<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QWidget" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>582</width>
     <height>388</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>580</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>580</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>SOURCE CODE FILES</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="filesWidget">
      <property name="minimumSize">
       <size>
        <width>580</width>
        <height>250</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>580</width>
        <height>250</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="addBtn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lnameLineEdit">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="reviseBtn">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>REVISE</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nrFilesEdit">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="removeBtn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="nrFilesLabel">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>No. of files</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections/>
</ui>
